src/API/Rest/Enums.cs

```diff
@@ -76,6 +76,11 @@ public enum InputInterfaceType
     /// </summary>
     public enum InferenceRequestType
     {
+        /// <summary>
+        /// Unknown request type
+        /// </summary>
+        Unknown,
```

Comment:
Is this used to meaning unsupported and cannot be processed? If we only know how to handle the defined ones, we should limit to the supported ones only, so the request pre-processor can reject the request right away.
Still remember the days using C/C++, a "supported/unknown" enum will be added at the end to use as an sentinel, but for C#, I don't think there is a need.

Reply:
This is reserved for internal use.